packages/ia-components/components/theatres/with-youtube-spotify/utils/get-track-list-by-source.js

```diff
@@ -9,12 +9,9 @@ const formatTime = (lengthInSeconds) => {
   const seconds = ((lengthInMS % 60000) / 1000).toFixed(0); // becomes a string, thanks JS math
   const extraMinute = seconds === '60';
   const secondsDisplay = extraMinute ? '00' : (seconds < 10 ? '0' : '') + seconds;
-  let minutesDisplay = minutes;
-  if (!minutesDisplay) {
-    minutesDisplay = '00';
-  }
+  let minutesDisplay = minutes ? `${minutes}` : '00';
```

Comment:
One minor update to this that I didn't realize in the other comment is that you're using the falsey check to check if `minutes` is `0`, correct (ie not `undefined`)? 

I might update this check to be `minutes === 0` if so. The `minutes + 1` below got me wondering if `minutes` could be `undefined` because of the falsey check.
```suggestion
  let minutesDisplay = minutes === 0 ? `${minutes}` : '00';
```